src/components/ClaimTask.jsx

```diff
@@ -19,6 +19,28 @@ const ClaimTask = props => {
     }
   }
 
+  const claimTask = async event => {
+    let headers = JSON.parse(localStorage.getItem('J-tockAuth-Storage'))
+    let id = event.target.parentElement.dataset.id
+    try {
+      let response = await axios.put(
+        `/tasks/${id}`,
+        { activity: 'claimed' },
+        { headers: headers }
+      )
+      if (response.status === 200)
+        dispatch({
+          type: 'GREETING',
```

Comment:
GREETING?

Reply:
It's a artifact of ancient days :D we can change it in a chore if you want?